HousingRepairsOnlineApi/UseCases/BookAvailableAppointmentUseCase.cs

```diff
@@ -35,7 +36,13 @@ public async Task Execute(string bookingReference, string sorCode, string priori
             {
                 throw new InvalidOperationException($"No appointments returned from Appointments Gateway");
             }
-            await appointmentsGateway.BookAppointment(bookingReference, sorCode, priority, locationId, appointment.TimeOfDay.EarliestArrivalTime, appointment.TimeOfDay.LatestArrivalTime, repairDescriptionText);
+
+            var communalRepairCommentText = repairDescriptionText + " " + repairDescriptionLocation;
```

Comment:
I feel a new line _could_ be a better separator as it would be more obvious where the repair description text ends and location begins.

But, I also think that having the location before the text is better too as it will provide the reader a location context prior to the other description.
It would be best to get a direction from the product owner on this.